src/components/Layout/RightPaneLayout/useRightPane.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode, useCallback } from "react";
+import { useRightPaneContext } from "./RightPaneContext";
+
+interface OpenRightPaneOpts {
+  content: ReactNode;
+}
```

Comment:
**suggestion:** This `OpenRightPaneOpts` interface on what properties are accepted by the `openInPane` are duplicated between here and in the Context file. Is there a reason to separate them out, or can we consolidate this?